modules/appeals_api/app/controllers/appeals_api/higher_level_reviews/v0/higher_level_reviews_controller.rb

```diff
@@ -64,23 +90,31 @@ def download
 
     def header_names = headers_schema['definitions']['hlrCreateParameters']['properties'].keys
 
+    def validate_json_schema
+      begin
+        validate_headers(request_headers)
+        validate_form_data(@json_body)
+      rescue JsonSchema::JsonApiMissingAttribute => e
+        render json: e.to_json_api, status: e.code
+      end
+
+      status, error = AppealsApi::HigherLevelReviews::PdfFormFieldV2Validator.new(@json_body, headers).validate!
+      return if error.blank?
+
+      render status:, json: error
+    end
+
+    def request_headers
+      header_names.index_with { |key| request.headers[key] }.compact
+    end
+
     def render_higher_level_review(hlr, **)
       render(json: HigherLevelReviewSerializer.new(hlr).serializable_hash, **)
     end
 
     def render_higher_level_review_not_found(id)
-      render(
-        status: :not_found,
-        json: {
-          errors: [
-            {
-              code: '404',
-              detail: I18n.t('appeals_api.errors.not_found', type: 'HigherLevelReview', id:),
-              status: '404',
-              title: 'Record not found'
-            }
-          ]
-        }
+      raise Common::Exceptions::ResourceNotFound.new(
```

Comment:
In these changes, I took the opportunity to replace our hand-crafted 404 errors with the reusable equivalent from `Common::Exceptions` - this does not change the behavior, but it does slightly alter the wording of the response - you can see those changes in the updated swagger.json files in this PR